packages/widgets/Source/Viewer/Viewer.js

```diff
@@ -1719,12 +1719,18 @@ Viewer.prototype.isDestroyed = function () {
 Viewer.prototype.destroy = function () {
   let i;
 
-  this.screenSpaceEventHandler.removeInputAction(
-    ScreenSpaceEventType.LEFT_CLICK
-  );
-  this.screenSpaceEventHandler.removeInputAction(
-    ScreenSpaceEventType.LEFT_DOUBLE_CLICK
-  );
+  // Check if Event handler is exist
+  if(define(this.screenSpaceEventHandler)){
```

Comment:
```suggestion
  if(defined(this.screenSpaceEventHandler)){
```

Reply:
CI is also failing due to formatting issues. Can you run `npm run prettier` and submit the result?